add summoner name and summoner id properties to LeagueItem model.

ArthurFSharp opened the pull request:
Add these two fields so that GetChallengerLeagueAsync request for example fill with datas.

```
RiotClient.DefaultPlatformId = PlatformId.EUW1;
            RiotClient.DefaultSettings = () => new RiotClientSettings
            {
                ApiKey = "" // Replace this with your API key, of course.
            };
            IRiotClient client = new RiotClient();

            var challengers = await client.GetChallengerLeagueAsync(RankedQueue.RANKED_SOLO_5x5);
```

Riot's API response model : 

```
{
    "tier": "CHALLENGER",
    "leagueId": "40ac4f31-647b-3960-b95e-45a8ee1ef734",
    "entries": [
        {
            "summonerName": "Whiplash76",
            "hotStreak": false,
            "wins": 167,
            "veteran": false,
            "losses": 79,
            "rank": "I",
            "inactive": false,
            "freshBlood": false,
            "summonerId": "RBfxKM0fn9hJ13T8HaAY9G5S3AdziHdjrrUEXQqEDNAsF4TF",
            "leaguePoints": 451
        },
```

Commit:
add summoner name and summoner id properties to LeagueItem model.

aj-r reviewed (changes requested):
Please update `RiotNet.Tests/Properties/Resources.resx` to use `summonerId` and `summonerName` instead of `playerOrTeamId` and `playerOrTeamName` (it might be easiest to open it as a text file and find/replace).

Also, should we delete `PlayerOrTeamId` and `PlayerOrTeamName` (or at least mark them as `[Obsolete]`) if they don't exist anymore?

Also make sure that the tests in `LeagueTests.cs` pass.

Commit:
Added position field

Commit:
Unit tests for summoner name and position field

ArthurFSharp commented:
I did what you asked me to do as a change.
I added the Position field to the LeaguePosition class and edited unit test. 

I removed playerOrTeamName and playerOrTeamId properties, they are not used anymore (since API v3 is obsolete right now). 

Some tests fail like GetChallengerLeagueAsyncTest because there is no challenger ladder for now : 

`Assert.That(league.Entries.Count, Is.GreaterThan(1));`

And I also ignored the tests for champions knowing that the champions APIs do not exist in the V4 (May be in a future change fetch results via the static APIs (Ddragon).).

aj-r reviewed (approved):
Ok, if there are unrelated failing tests that's fine.

Thanks for your contribution!
